installer/windows/microk8s.nsi

```diff
@@ -0,0 +1,175 @@
+        "Software\Microsoft\Windows\CurrentVersion\Uninstall\${PRODUCT_NAME}" \
+        "UninstallString" "$INSTDIR\uninstall.exe"
+SectionEnd
+
+Section "Add 'microk8s' to PATH" add_to_path_id
+    EnVar::AddValue "path" "$INSTDIR"
+SectionEnd
+
+!insertmacro MUI_FUNCTION_DESCRIPTION_BEGIN
+    !insertmacro MUI_DESCRIPTION_TEXT ${multipass_id} "REQUIRED: If already installed, will be unticked and skipped.$\n$\nSee https://multipass.run for more."
+    !insertmacro MUI_DESCRIPTION_TEXT ${add_to_path_id} "Add the 'microk8s' executable to PATH.$\n$\nThis will allow you to run the command 'microk8s' in cmd and PowerShell in any directory."
+!insertmacro MUI_FUNCTION_DESCRIPTION_END
+
+Function .onInit
+    DeleteRegKey HKLM "Software\canonical\multipass"
+    IfFileExists $PROGRAMFILES64\Multipass\bin\multipass.exe untickMultipass tickMultipass
+    tickMultipass:
+        IntOp $0 ${SF_SELECTED} | ${SF_RO}
+        SectionSetFlags ${multipass_id} $0
+        Return
+    untickMultipass:
+        SectionSetFlags ${multipass_id} ${SF_RO}
+        Return
+FunctionEnd
+
+Function "ConfigureVm"
+    ClearErrors
+    ReadRegStr $0 HKLM "Software\canonical\multipass" "InstallerRebootRequired"
+    ${IfNot} ${Errors}  ; Error means hypervisor is available, so we can proceed.
+        ${If} $0 == "true"
+            MessageBox MB_ICONEXCLAMATION "Cannot configure the ${PRODUCT_NAME} VM as a reboot is required.  Please re-run this wizard after reboot to configure the VM."
+            SetRebootFlag true
+            Abort
+        ${ElseIf} $0 == "false"
+            MessageBox MB_ICONEXCLAMATION "Cannot configure the ${PRODUCT_NAME} VM as VirtualBox needs to be installed.  Please re-run this wizard after installing VirtualBox to configure the VM."
+            Abort
+        ${EndIf}
+    ${EndIf}
+    MessageBox MB_YESNO "Do you want to configure and launch the ${PRODUCT_NAME} VM now?" /SD IDYES IDNO endLaunch
+        nsDialogs::Create 1018
+        Pop $VmConfigureDialog
+
+        ${NSD_CreateLabel} 19% 0 35u 10u "CPUs"
+        Pop $VmConfigureDialogCpuLabel
+
+        ${NSD_CreateLabel} 44% 0 35u 10u "Mem (GB)"
+        Pop $VmConfigureDialogMemLabel
+
+        ${NSD_CreateLabel} 69% 0 35u 10u "Disk (GB)"
+        Pop $VmConfigureDialogDiskLabel
+
+        ${NSD_CreateNumber} 19% 17.5 35u 10u "2"
+        Pop $VmConfigureDialogCpu
+
+        ${NSD_CreateNumber} 44% 17.5 35u 10u "4"
+        Pop $VmConfigureDialogMem
+
+        ${NSD_CreateNumber} 69% 17.5 35u 10u "50"
+        Pop $VmConfigureDialogDisk
+
+        ${NSD_CreateLabel} 42% 50 50u 10u "Snap Track"
+        Pop $VmConfigureDialogTrackLabel
+
+        ${NSD_CreateText} 42% 67.5 50u 10u "1.18/stable"
+        Pop $VmConfigureDialogTrack
+
+        ${NSD_CreateLabel} 8% 102.5 100% 10u "These are the minimum recommended parameters for the VM running ${PRODUCT_NAME}"
+        Pop $VmConfigureDialogFootnote
+
+        nsDialogs::Show
+    endLaunch:
+        Abort
+FunctionEnd
```

Comment:
Consider the following layout:
![VM-conf-layout](https://user-images.githubusercontent.com/11489452/82655109-5f522580-9c2a-11ea-9310-9799c22c6ee7.png)
```
Function "ConfigureVm"
    ClearErrors
    ReadRegStr $0 HKLM "Software\canonical\multipass" "InstallerRebootRequired"
    ${IfNot} ${Errors}  ; Error means hypervisor is available, so we can proceed.
        ${If} $0 == "true"
            MessageBox MB_ICONEXCLAMATION "Cannot configure the ${PRODUCT_NAME} VM as a reboot is required.  Please re-run this wizard after reboot to configure the VM."
            SetRebootFlag true
            Abort
        ${ElseIf} $0 == "false"
            MessageBox MB_ICONEXCLAMATION "Cannot configure the ${PRODUCT_NAME} VM as VirtualBox needs to be installed.  Please re-run this wizard after installing VirtualBox to configure the VM."
            Abort
        ${EndIf}
    ${EndIf}
    MessageBox MB_YESNO "Do you want to configure and launch the ${PRODUCT_NAME} VM now?" /SD IDYES IDNO endLaunch
        nsDialogs::Create 1018
        Pop $VmConfigureDialog

        ${NSD_CreateLabel} 0% 25u 100% 12u "Please configure the VM running MicroK8s"
        Pop $VmConfigureDialogFootnote
        ${NSD_AddStyle} $VmConfigureDialogFootnote ${SS_Center}


        ${NSD_CreateLabel} 30% 40u 20% 10u "CPU (cores): "
        Pop $VmConfigureDialogCpuLabel
        ${NSD_AddStyle} $VmConfigureDialogCpuLabel ${SS_Right}

        ${NSD_CreateNumber} 50% 40u 25u 10u "2"
        Pop $VmConfigureDialogCpu



        ${NSD_CreateLabel} 30% 55u 20% 10u "Mem (GB): "
        Pop $VmConfigureDialogMemLabel
        ${NSD_AddStyle} $VmConfigureDialogMemLabel ${SS_Right}

        ${NSD_CreateNumber} 50% 55u 25u 10u "4"
        Pop $VmConfigureDialogMem



        ${NSD_CreateLabel} 30% 70u 20% 10u "Disk (GB): "
        Pop $VmConfigureDialogDiskLabel
        ${NSD_AddStyle} $VmConfigureDialogDiskLabel ${SS_Right}

        ${NSD_CreateNumber} 50% 70u 25u 10u "50"
        Pop $VmConfigureDialogDisk



        ${NSD_CreateLabel} 30% 85u 20% 10u "Snap Track: "
        Pop $VmConfigureDialogTrackLabel
        ${NSD_AddStyle} $VmConfigureDialogTrackLabel ${SS_Right}

        ${NSD_CreateText} 50% 85u 85u 10u "1.18/stable"
        Pop $VmConfigureDialogTrack


        nsDialogs::Show
    endLaunch:
        Abort
FunctionEnd
```